app/views/shared/_posts_container.erb

```diff
@@ -0,0 +1,9 @@
+<div class="usersPosts">
+<% @user.recent_posts.each do |post| %> 
+    <div class="post">
+      <h4><%= link_to post.title, user_posts_path(user_id: post.author_id), class: 'custom-link-s' %></h4>
```

Comment:
- [ ] I suggest that clinking on a post title links to the post-show page(`user_post_path`), not the posts index page (`user_posts_path`) :blush:

Reply:
Great idea @Owusu-Desmond !! 

done!